### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by soyfrien**
apt provides a high-level commandline interface for the package management system. It is intended as an end user interface and enables some options better suited for interactive usage by default compared to more specialized APT tools like apt-get...
https://manpages.ubuntu.com/manpages/focal/en/man8/apt.8.html

### CODE REVIEW

**Review by Jimbo4350 — APPROVED** [review]
Thanks :+1:

**Comment by Jimbo4350** [comment]
@soyfrien Can you rebase this on master? I'll merge it after.

**Comment by soyfrien** [comment]
Apologies, if I did that incorrectly, I think we can just close this and edit the current branch. Very sorry for the delay.

**Comment by soyfrien** [comment]
Sorry, I'll figure this out...

**Comment by soyfrien** [comment]
Hi @Jimbo4350, thank you for the advice and patience for small issue. I wound up giving into ask for help, the git docs are challenging.

[
Was this the proper advice?](https://stackoverflow.com/a/71583547/434034)

Just want to know if I have us stuck again.

File: doc/getting-started/install.md [hunk 1]
```diff
@@ -39,12 +39,13 @@ sudo yum install systemd-devel ncurses-devel ncurses-compat-libs -y
 
 For Debian/Ubuntu, use the following instead:
 
-```bash
-sudo apt-get update -y
-sudo apt-get install automake build-essential pkg-config libffi-dev libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev make g++ tmux git jq wget libncursesw5 libtool autoconf -y
+
```
Comment (Jimbo4350) [inline doc/getting-started/install.md:42]: @soyfrien Why have you removed ```bash?

Comment (soyfrien) [reply doc/getting-started/install.md:42]: Oops, I am going to close this, it's many commits behind now and it's not a change that makes a difference to any outcomes. I'll add it in if I ever make a bigger pull request.

**Comment by soyfrien** [comment]
Closing to help free up the queue.

**Pull request closed without merging**